### PR TITLE
Create LIT6465PaterNostCoptBoh.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New entry.  How can it be linked to ETH1292Bohair? Shoult they be linked?

### CODE REVIEW

**Comment by eu-genia** [comment]
do we create a work record for each language version? I thought our rule was one work one record, languages are distinguished by title ids

**Comment by thea-m** [comment]
I would also tend towards sticking with the existing ID LIT2168PaterN. If you agree with us, this PR has to be merged and a new PR created for the deletion of the new ID. Otherwise, I think this discussion should better be transferred to an issue.

**Comment by eu-genia** [comment]
yes, cp. also my comment here https://github.com/BetaMasaheft/Manuscripts/pull/1149#discussion_r654632874

File: 2001-3000/LIT2168PaterN.xml [hunk 1]
```diff
@@ -1,11 +1,14 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT2168PaterN">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
             <title xml:id="t1">Pater Noster</title>
             <title xml:id="t2">Lord’s Prayer, Our Father</title>
```
Comment (eu-genia) [inline 2001-3000/LIT2168PaterN.xml:8]: lord's prayer and our father are two separate titles
I suggest
```
<title xml:id="t1" xml:lang="lat">Pater Noster</title>
<title corresp="#t1" xml:lang="en">Our Father</title>
<title xml:id="t2" xml:lang="en">Lord's Prayer</title>
```

File: 2001-3000/LIT2168PaterN.xml [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+              <title xml:lang="gez" corresp="#t3" xml:id="t3">አቡነ፡ ዘበሰማያት፡</title>
+              <title xml:lang="gez" corresp="#t3" type="normalized">ʾAbuna zabasamāyāt</title>
```
Comment (eu-genia) [inline 2001-3000/LIT2168PaterN.xml:10]: I would supply language to title IDs, t3geez and add translation

`<title corresp="#t3" xml:lang="en">Our Father in Heaven</title>
`

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

**Comment by thea-m** [comment]
I think that the correct workflow in such cases is to create the superfluous record and then make a new PR for the deletion...

**Pull request merged**